src/Servers/Kestrel/Transport.Sockets/src/Internal/IOQueue.cs

```diff
@@ -73,4 +75,19 @@ public Work(Action<object?> callback, object? state)
             State = state;
         }
     }
+
+    private static int DetermineDefaultCount()
+    {
+        // Since each IOQueue schedules one work item to process its work, the number of IOQueues determines the maximum
+        // parallelism of processing work queued to IOQueues. The default number below is based on the processor count and tries
+        // to use a high-enough number for that to not be a significant limiting factor for throughput.
+
+        int processorCount = Environment.ProcessorCount;
```

Comment:
Nit:
```suggestion
        var processorCount = Environment.ProcessorCount;
```

Reply:
Isn't the explicit type preferred in this case? I thought that was the case in the runtime repo anyway, not sure about the guidelines in this repo.

Reply:
Even if var is allowed in this repo, how is it better here?

Reply:
"`var` all the things" is convention here (personally I prefer the explicit type, but to be consistent w/ the codebase -> `var`).